### PR TITLE
Improved PVR scrobbler

### PR DESCRIPTION
**Pull request by Lunatixz**
PVR channels with proper tags were not scrobbling, corrected infolabels
and added PVR progress tracking.

### CODE REVIEW

**Comment by Lunatixz** [comment]
@Razzeee issue with this PR, Scrobbler stops after initial title. I'm currently looking for a place to drop in a check.

**Comment by Lunatixz** [comment]
Looks like onAVChange() should do the trick...

File: addon.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="script.trakt" name="Trakt" version="3.2.4" provider-name="Trakt.tv, Razzeee">
+<addon id="script.trakt" name="Trakt" version="3.2.4b" provider-name="Trakt.tv, Razzeee">
```
Comment (razzeee) [inline addon.xml]: Can you remove this for now? I like to keep code and the version bumps separated :)

Comment (Lunatixz) [reply addon.xml]: Done

**Pull request merged**